rpcserver/rpcserver.h

```diff
@@ -19,4 +19,6 @@ class RPCServer {
         bool receive_handshake(tcp::socket& sock, short* version, short* service, short* auth_protocol);
```

Comment:
@pelmers , although only 2 ppl ever work on this so maybe its fine to leave undocumented, in the interest of time :)